src/map/packets/currency1.cpp

```diff
@@ -153,6 +154,14 @@ CCurrencyPacket1::CCurrencyPacket1(CCharEntity* PChar)
         ref<uint8>(0xD6) = _sql->GetUIntData(80); // rems_ch9
         ref<uint8>(0xD7) = _sql->GetUIntData(81); // rems_ch10
 
+        // Salvage II Stored Plans
+        ref<uint64>(0xD8) = PackSalvageIIPlans(
```

Comment:
you can simplify this by defining an enum class defined in this cpp file:

```
// TODO: implement the packet wholesale
// from https://github.com/atom0s/XiPackets/tree/main/world/server/0x0113
struct bitPackedCurrency1
{
    uint64_t bloodshed_plans_stored   : 9;
    uint64_t umbrage_plans_stored     : 9;
    uint64_t ritualistic_plans_stored : 9;
    uint64_t tutelary_plans_stored    : 9;
    uint64_t primacy_plans_stored     : 9;
    uint64_t unused                   : 19;
};
```

Make sure to co-author atom0s by putting this in your commit text:
`Co-authored-by: atom0s <atom0s@live.com>`

Reply:
> you can simplify this by defining an enum class defined in this cpp file:
> 
> ```
> // TODO: implement the packet wholesale
> // from https://github.com/atom0s/XiPackets/tree/main/world/server/0x0113
> struct bitPackedCurrency1
> {
>     uint64_t bloodshed_plans_stored   : 9;
>     uint64_t umbrage_plans_stored     : 9;
>     uint64_t ritualistic_plans_stored : 9;
>     uint64_t tutelary_plans_stored    : 9;
>     uint64_t primacy_plans_stored     : 9;
>     uint64_t unused                   : 19;
> };
> ```
> 
> Make sure to co-author atom0s by putting this in your commit text: `Co-authored-by: atom0s <atom0s@live.com>`

I've added the struct and updated the code to use it. I also made sure to credit @atom0s <3